crates/forge/tests/cli/config.rs

```diff
@@ -205,6 +205,7 @@ forgetest_init!(can_override_config, |prj, cmd| {
     );
 
     // env vars work
+    std::env::remove_var("DAPP_REMAPPINGS");
```

Comment:
we should just not be using env vars in cli tests, it can only work if its set on a spawned command

Reply:
yeah, will clean those tests and move such remapping test in dedicated